### PR TITLE
Nuke

### PR DESCRIPTION
**Pull request by eaba**
## Checklist

Build system is ancient here - need to bring over the https://github.com/petabridge/Petabridge.Library/ build engine.

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
@eaba does this work correctly locally?

**Comment by eaba** [comment]
Not yet

**Comment by eaba** [comment]
continuous-integration/teamcity @Aaronontheweb

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
@eaba there's an enormous number of binary files committed in this PR that should not be.  - all under the ./build/tools directory.

**Comment by eaba** [comment]
@Aaronontheweb

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Comment by mchandschuh** [comment]
Thanks for the quick turn around guys! 🙇

File: src/Akka.TestKit.NUnit/TestKit.cs [hunk 1]
```diff
@@ -15,6 +15,9 @@ namespace Akka.TestKit.NUnit
     /// <summary>
     /// TestKit for NUnit.
     /// </summary>
+    /// 
+    [TestFixture]
+    [FixtureLifeCycle(LifeCycle.InstancePerTestCase)]
```
Comment (mchandschuh) [inline src/Akka.TestKit.NUnit/TestKit.cs:20]: @eaba  This is my fault -- I overlooked this required change in the initial PR. We need to remove this throw [here](https://github.com/akkadotnet/Akka.TestKit.NUnit/pull/47/commits/ae5a6e00ac19e8224a17d1b99538a3cac5809160#diff-835e2f8d56950585d517422fc80535cf17ae703f07536b0850e66b3f1fc256f1R38) and [here](https://github.com/akkadotnet/Akka.TestKit.NUnit/pull/47/commits/ae5a6e00ac19e8224a17d1b99538a3cac5809160#diff-b9804ef5e8a84dab03c76aed0478fc4e0fbaa54f30f26ac90b2cd34b6110d644R38)

The `[FixtureLifeCycle(LifeCycle.InstancePerTestCase)]` makes it so we can pass an `ActorSystem`. Here's the original [PR](https://github.com/akkadotnet/akka.net/pull/1092) that added the _code bomb_ per @Aaronontheweb

Comment (mchandschuh) [reply src/Akka.TestKit.NUnit/TestKit.cs:20]: A new unit test showing that we can pass `ActorSystem` to `TestKit.ctor` would be nice as well.

**Pull request merged**